### PR TITLE
Extract Landing page from EmbarkRoot

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Extract Landing page from EmbarkRoot
- Move entry animation directly to Landing page component.
- Remove "showLanding" prop from EmbarkRoot.
- Show Landing for path /new-member.

> The diff looks more complicated than it is. Here you can view without whitespace changes: https://github.com/HedvigInsurance/web-onboarding/pull/523/files?diff=split&w=1

## Why?

Seems cleaner to separate these pages base on the route. EmbarkRoot doesn't need to be concerned about rendering Landing page.

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -170,12 +171,7 @@ export const reactPageRoutes: ReactPageRoute[] = [
   {
     path: LOCALE_PATH_PATTERN + '/new-member',
     render: ({ match }: RouteComponentProps<any>) => (
-      <EmbarkRoot
-        language={match.params.locale}
-        name={undefined}
-        baseUrl={undefined}
-        showLanding={true}
-      />
+      <Landing language={match.params.locale} />
```
Comment (styrelius) [inline src/routes.tsx:174]: We could also remove the `language` prop here and replace it with using `useCurrentLocale` inside the `Landing` component where `language` is used now (which is only one place) -  might make this even more clean 😀

**Review by styrelius — COMMENTED** [review]
Super nice 👌 ✨ 

One suggestion that might improve this even further 👇

File: src/routes.tsx (hunk 1 above)
Comment (robinandeer) [reply src/routes.tsx:174]: Aha, `language` and `currentLocale` are the same things!

**Comment by styrelius** [comment]
@robinandeer Maybe you've already investigated this but the reason bor the failed build is that type checking fails since we also need to remove the `language` prop from the `Landing` component in `Landing.stories.tsx` 😄

**Comment by robinandeer** [comment]
> @robinandeer Maybe you've already investigated this but the reason bor the failed build is that type checking fails since we also need to remove the `language` prop from the `Landing` component in `Landing.stories.tsx` 😊

Thanks -- good reminder to run test/lint locally 🙃

**Review by styrelius — APPROVED** [review]
This is great! 🤩  Makes so much more sense that what we had previously. Checked locally that the session token still was created and stored as it should and worked fine to go through the flow, get an offer and sign.

It also seems to work with the apps - I could sign after going through the flow with query param `variation=IOS`, and the signed member could be found in Hope (had some issues with this last time I checked but I probably did something wrong then since it's working totally fine now)

**Comment by gustaveen** [comment]
> This is great! 🤩 Makes so much more sense that what we had previously. Checked locally that the session token still was created and stored as it should and worked fine to go through the flow, get an offer and sign.
> 
> It also seems to work with the apps - I could sign after going through the flow with query param `variation=IOS`, and the signed member could be found in Hope (had some issues with this last time I checked but I probably did something wrong then since it's working totally fine now)

Awesome, then I'm all for it! 🎉

**Review by gustaveen — APPROVED** [review]

**Comment by robinandeer** [comment]
Woho! I'll leave this open until we go live with the current staging build to have that to roll back to JIC

**Pull request merged**